Cookbook/Technical-Documents/XcodeTips.md

```diff
@@ -0,0 +1,11 @@
+## Tips and tricks
+
+### Errors when pulling code/switching branches
+Various errors can occur when pulling code/switching branches. Typically all that is needed is running `pod install` and closing and reopening the project in XCode. The following directions are intended to be a more thorough reset to deal with bigger changes.
+
+- Clean Build: Product > Clean (shortcut: cmd + shift + k)
+- Close Xcode
+- Update bundle: run `bundle install`
+- Update pods: `pod repo update && pod deintegrate && pod install`
+- Clean DerivedData: can be found at ~/Library/Developer/Xcode/DerivedData and/or babylon-ios/DerivedData
```

Comment:
I doubt it will ever be in `babylon-ios/DerivedData` unless some one configured it for themselves. Anyway `Option-Cmd-Shift-K` should do the job (I always run it with `Cmd-Shift-K`)

Reply:
I've been seeing it pop up in both locations for some reason. Still not sure why (don't think I did anything to configure as such, and it still populates normally in ~Library/Developer/Xcode). So for now, I'd like to keep it here just in case others run into the same thing.

Reply:
my only guess is that it can be possible if one makes a build with fastlane, but otherwise it is not used when building with Xcode

Reply:
That would make sense. I had been exploring those commands around that same time.